tastypie/__init__.py

```diff
@@ -3,7 +3,9 @@
 
 __author__ = 'Daniel Lindsley & the Tastypie core team'
 
-VERSION = (0, 14, 2)
+# This variable should include the Django-tastypie version number (indexes 0,1,2) and the Grok version number for our fork (from index 3).
+# This is required so that we have a distinct version number for each internal Grok version of Django-tastypie, so pip knows to update tastypie when running the app repo.
+VERSION = (0, 14, 2, '-grokfork-0.1')
```

Comment:
I think you want to use a local version identifier - see https://www.python.org/dev/peps/pep-0440/#local-version-identifiers

i.e. the final version would be something like 0.14.2+grokfork-0.1

Reply:
Cool. That would also mean it has to be:
`0.14.2+grokfork.0.1`

As it specifies local version identifiers must be restricted to:
- ASCII letters ([a-zA-Z])
- ASCII digits ([0-9])
- periods (.)